packages/providers/src/transaction-response/transaction-response.ts

```diff
@@ -91,7 +91,11 @@ export class TransactionResponse {
   /** Gas used on the transaction */
   gasUsed: BN = bn(0);
   /** Number off attempts to get the committed tx */
-  attempts: number = 0;
+  fetchAttempts: number = 0;
+
+  resultAttempts: number = 0;
+
+  gqlTransaction?: GqlTransaction;
```

Comment:
Please can we get some commented descriptions on these as typedoc is exposing this class